README.md

```diff
@@ -25,6 +25,8 @@ Simple, Pythonic building blocks to evaluate LLM applications.
 pip install langcheck
 ```
 
+Having installation issues? [See the FAQ](https://langcheck.readthedocs.io/en/latest/installation.html#installation-faq).
```

Comment:
@liwii can you add this link in README_ja.md as well?

Reply:
Added the link to the English FAQ.